pwa/src/apiService/resources/organizations.ts

```diff
@@ -0,0 +1,59 @@
+import { Send } from "../apiService";
```

Comment:
A resource filename (and the classname, but that's already correct) should be singular.